config/owasp/suppressions.xml

```diff
@@ -24,4 +24,10 @@
     <cve>CVE-2020-8022</cve>
     <cve>CVE-2017-18640</cve>
   </suppress>
+
+  <!--Added on 20th May 2020-->
```

Comment:
Do not to add new CVEs unless there is no solution for it at the moment.

Reply:
Not sure about these CVEs, they exists in last change, But will see if they are valid with recent CVE fixes.

Reply:
I have checked the CVEs CVE-2017-18640, CVE-2020-8022 we still need to suppress as we are using the following Jars,
snakeyaml-1.25.jar
tomcat-embed-core-9.0.31.jar
tomcat-embed-websocket-9.0.31.jar

Reply:
What I mean is to try to upgrade the dependencies and resolve the CVE error instead of suppressing them

Reply:
These tomcat & snake libs are transitive dependencies of Spring Boot
I think we upgraded Spring Boot for All the Apps.